vs.proj/mimeTools.vcxproj

```diff
@@ -107,13 +107,15 @@
     <ClCompile>
       <PrecompiledHeader>NotUsing</PrecompiledHeader>
       <WarningLevel>Level4</WarningLevel>
-      <Optimization>Full</Optimization>
+      <Optimization>Disabled</Optimization>
```

Comment:
Why is it disabled?

Reply:
@donho Because it is the debug configuration and typically there you want no optimizations to be able to debug the plugin properly.
In the meantime vs2015 is already a bit outdated. I will update the appveyor config and this PR.

Reply:
It makes sense.